nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -0,0 +1,324 @@
+import numpy as np
```

Comment:
imports needs to be reordered. More general to local.
import os
import tempfile

import nibabel
......